app/helpers/dashboard_helper.rb

```diff
@@ -0,0 +1,40 @@
+module DashboardHelper
+  def top_three_companies_by_interest_level
```

Comment:
do you have any ideas on where this method might live that's more appropriate than a helper module?